Importer/CsvImporterFactory.php

```diff
@@ -68,6 +69,8 @@ public function create(ConfigurationInterface $configuration, $delimiter = ',')
         $log = $this->logFactory->create();
         $interpreter = $this->dataInterpreterFactory->create($configuration, $log);
 
-        return new CsvImporter($configuration, $interpreter, $this->entityManager, $this->csvParser, $log, $this->eventDispatcher, $delimiter);
+        $errorExceptionEventFactory = new ErrorExceptionEventFactory($log);
```

Comment:
>Same logic we follow on EntityDataInterpreterFactory. 

Perhaps that shows another place where we need refactoring. Just because it is ugly in one place doesn't mean we can repeat it everywhere 😄 .